controllers/create_indexerconfigmap.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Contributors to the Open Cluster Management project
+package controllers
+
+import (
+	"context"
+
+	cachev1 "github.com/stolostron/search-v2-operator/api/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *OCMSearchReconciler) createIndexerConfigmap(request reconcile.Request,
```

Comment:
This function is missing the case where the configmap already exists.  Could we check that the current contents match the expected content?  If it doesn't match, should we change update it or just report the error?

I see this same pattern in several files, maybe we should make it a common function?

Reply:
Yes - thats coming